fixtures/vcr_cassettes/test_uprn_view.yaml

```diff
@@ -75,8 +74,7 @@ interactions:
         Council","email":"elections@shropshire.gov.uk","phone":"0345 678 9015","website":"http://www.shropshire.gov.uk/","postcode":"SY2
         6ND","address":"Electoral Registration Officer\nShirehall\nAbbey Foregate\nShrewsbury","identifiers":["E06000051"],"nation":"England"},"registration":{"council_id":"SHR","name":"Shropshire
         Council","email":"elections@shropshire.gov.uk","phone":"0345 678 9015","website":"http://www.shropshire.gov.uk/","postcode":"SY2
-        6ND","address":"Electoral Registration Officer\nShirehall\nAbbey Foregate\nShrewsbury","identifiers":["E06000051"],"nation":"England"},"postcode_location":{"type":"Feature","properties":null,"geometry":{"type":"Point","coordinates":[-2.326615276923077,52.49552346923076]}},"parl_boundary_changes":{"current_constituencies_official_identifier":"gss:E14000799","current_constituencies_name":"Ludlow","new_constituencies_official_identifier":"gss:E14001493","new_constituencies_name":"South
-        Shropshire","CHANGE_TYPE":"NAME_CHANGE_BOUNDARY_CHANGE"}}'
+        6ND","address":"Electoral Registration Officer\nShirehall\nAbbey Foregate\nShrewsbury","identifiers":["E06000051"],"nation":"England"},"postcode_location":{"type":"Feature","properties":null,"geometry":{"type":"Point","coordinates":[-2.326615276923077,52.49552346923076]}}}'
```

Comment:
I just manually edited the JSON in these test fixtures.
Do you have a better coping strategy for re-building them when the underlying data in the upstream API has changed?
In general, I feel like VCR is a poor approach for mocking our API responses because any given postcode/UPRN is going to return totally different data depending on when you request it.

Reply:
Yep, VCR isn't the right thing to use at all. See conversations about response builder!